src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -390,7 +390,7 @@ public function load(array $configs, ContainerBuilder $container)
         if ($this->readConfigEnabled('mailer', $container, $config['mailer'])) {
             $this->registerMailerConfiguration($config['mailer'], $container, $loader);
 
-            if (!class_exists(MailerTestCommand::class) || !$this->hasConsole()) {
+            if (!$this->hasConsole() || !class_exists(MailerTestCommand::class)) {
```

Comment:
Why does changing the order here make a difference?

Reply:
I'm... having a hard time figuring out why myself... @featuriz?